packages/components/Tabs/src/Tabs.types.ts

```diff
@@ -34,10 +34,20 @@ export interface TabsContextData {
    */
   tabsItemKeys?: string[];
 
+  /**
+   * Array of enabled keys in the group
+   */
+  enabledKeys?: string[];
+
   /**
    * A Map to store the corresponding view of each TabsItem
    */
   views?: Map<string, React.ReactNode[]> | null;
+
+  /**
+   * Reference to the Focus Container
+   */
+  focusZoneRef: React.RefObject<any> | null;
```

Comment:
I would update the comment here as well to say this is because we don't have focus zone on windows